pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -467,7 +476,15 @@ def _update_task_incoming(
         return db_document
 
     def _set_logs(self, payloads: dict, start_time: str) -> Dict:
-        """Set up the logs for the incoming request."""
+        """Set up the logs for the incoming request.
```

Comment:
Better (more precies): "Create or update `TesTask.logs` and set start time"

Reply:
Done